radis/io/hitran.py

```diff
@@ -690,12 +691,16 @@ def _parse_HITRAN_group1(df):
     )
     # ... note @EP: in HITRAN H2O files, for iso=2, the Kau, Kcu can somehow
     # ... be negative. The regex above is adapted to catch negation signs with \-
-
-    # 2. Convert to numeric
-    for k in ["ju", "Kau", "Kcu"]:
-        dgu[k] = pd.to_numeric(dgu[k])
-    for k in ["jl", "Kal", "Kcl"]:
-        dgl[k] = pd.to_numeric(dgl[k])
+    
+    try:
+        # 2. Convert to numeric
+        for k in ["ju", "Kau", "Kcu"]:
+            dgu[k] = pd.to_numeric(dgu[k], errors='coerce').astype(float64)
+        for k in ["jl", "Kal", "Kcl"]:
+            dgl[k] = pd.to_numeric(dgl[k], errors='coerce').astype(float64)
+
+    except:
```

Comment:
Do you know what exception was raised there? Maybe we need a more explicit print / verbose level?